subsys/caf/events/CMakeLists.txt

```diff
@@ -27,3 +27,7 @@ zephyr_sources_ifdef(CONFIG_CAF_LED_EVENTS
 zephyr_sources_ifdef(CONFIG_CAF_SENSOR_EVENTS
 	sensor_event.c
 )
+
+zephyr_sources_ifdef(CONFIG_CAF_BLE_COMMON_EVENTS
+	ble_common_event.c
+)
```

Comment:
I would put those in between `*_PM_EVENTS` and `*_BUTTON_EVENTS`. I assume `CONFIG_CAF_MODULE_STATE_EVENTS` and `CONFIG_CAF_PM_EVENTS` are so called "core" for CAF and the rest modules should be put in alphabetical order.

Reply:
ok, done